exercises/proverb/Sources/ProverbExample.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+struct ProverbExample {
+
+    func recite(pieces: [String]) -> String {
+
+    var saying = String()
+
+    if !pieces.isEmpty {
```

Comment:
Changing this to `if let first = pieces.first {` will also ensure that pieces is not empty, while avoiding the force unwrap below.